payload-schemas/api.github.com/common/package-npm-metadata.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "common/package-npm-metadata.schema.json",
+  "type": "object",
+  "properties": {
+    "name": { "type": "string" },
+    "version": { "type": "string" },
+    "npm_user": { "type": "string" },
+    "author": { "type": ["object", "null"] },
+    "bugs": { "type": ["object", "null"] },
+    "dependencies": { "type": "object" },
```

Comment:
I assume these map to `package.json` - is worth us trying to enhance this further because we know the type should be `Record<string, string>` for at least some of these right?

Reply:
This is how GitHub defines it themselves in their OpenAPI spec.

I wouldn't hurt to do that

Reply:
I've added `additionalProperties` https://github.com/octokit/webhooks/commits/8ece2ebe4416e808e98e8111d1e1e1c87f7ac925
Now they are `Record<string, string>`